app/src/pages/common/settingsPage/settingsPage.jsx

```diff
@@ -185,7 +186,12 @@ export class SettingsPage extends Component {
     if (!canSeeDemoData(this.props.accountRole, this.props.userRole)) {
       delete tabsConfig[DEMO_DATA];
     }
-    return { ...tabsConfig, ...this.createExtensionTabs() };
+    Object.keys(extensionTabs).forEach((tab) => {
+      if (tabsConfig[tab.name]) {
+        tabsConfig[tab.name].component = tab.component;
+      }
+    });
+    return { ...tabsConfig, ...extensionTabs };
```

Comment:
In this case items from `extensionTabs` will override items from `tabsConfig`.
So please swap them.
`{ ...extensionTabs, ...tabsConfig };`